scripts/export_large_tsv/export_large_tsv.py

```diff
@@ -82,36 +88,12 @@ def download_tsv_from_workspace(project, workspace, entity_type, tsv_name, page_
     # argument parser
     parser = argparse.ArgumentParser(description="Exports/downloadload a TSV file from Terra when it is too large to download via the UI.")
     # application arguments
-    parser.add_argument(
-        '--project',
-        type=str,
-        action='store',
-        required=True,
-        help='Terra namespace/project of workspace.')
-    parser.add_argument(
-        '--workspace',
-        type=str,
-        action='store',
-        required=True,
-        help='Name of Terra workspace.')
-    parser.add_argument(
-        '--entity_type',
-        type=str,
-        action='store',
-        required=True,
-        help='Entity type being requested for tsv export to local destination.')
-    parser.add_argument(
-        '--tsv_filename',
-        type=str,
-        action='store',
-        required=True,
-        help='Name of tsv file to be exported from Terra to local destination.')
-    parser.add_argument(
-        '--page_size',
-        type=int,
-        default=1000,
-        action='store',
-        help='Number of entities/rows to export per page.')
+    parser.add_argument('-p', '--project', type=str, required=True, help='Terra namespace/project of workspace.')
+    parser.add_argument('-w', '--workspace', type=str, required=True, help='Name of Terra workspace.')
+    parser.add_argument('-e', '--entity_type', type=str, required=True, help='Entity type being requested for tsv export to local destination.')
+    parser.add_argument('-f', '--tsv_filename', type=str, required=True, help='Name of tsv file to be exported from Terra to local destination.')
+    parser.add_argument('-n', '--page_size', type=int, default=1000, help='Number of entities/rows to export per page.')
+    parser.add_argument('-a', '--attribute_list', nargs='+', help='Number of entities/rows to export per page.')
```

Comment:
update argument description, with explanation of how the attributes should be included (e.g. comma separated, separate args? i think it's separate args, right? but is it like `-a attr1 attr2` or `-a attr1 -a attr2` ?)